### PR TITLE
Changes the elb_mapper to count tutorials, bucketing by type (e.g., begin, beginpng, 443).

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: aws/emr/cdo-logs/script/elb_mapper.rb [hunk 1]
```diff
@@ -84,13 +84,17 @@
...
   tutorials.each do |t|
-    if (line.include? "begin" + "\/" + t) ||
-      (line.include? "begin" + "_" + t) ||
-      (line.include? "443" + "\/" + t)
-      # Using LONG_VALUE_SUM instructs hadoop's streaming aggregate class how to
-      # aggregate. Using the date and tutorial as the key gives breakdowns by
-      # day and tutorial.
-      puts LONG_VALUE_SUM + date + " " + t + "\t" + "1"
+    # Using LONG_VALUE_SUM instructs hadoop's streaming aggregate class how to
+    # aggregate. Using the date and tutorial as the key gives breakdowns by
```
Comment (Hamms) [inline aws/emr/cdo-logs/script/elb_mapper.rb]: Could ya update this comment to mention that we're using date, tutorial, and type of metric as the key?

Comment (ashercodeorg) [inline aws/emr/cdo-logs/script/elb_mapper.rb]: Done.

**Comment by Hamms** [comment]
LGTM! Can I assume that the plan is to merge in #6139 first, then fix the conflicts in this branch before merging it in?

**Comment by ashercodeorg** [comment]
Closing because of merge conflicts, with PR #6179 (https://github.com/code-dot-org/code-dot-org/pull/6179) being the "merged" version.

**Pull request closed without merging**